.github/workflows/ci.yaml

```diff
@@ -47,6 +47,22 @@ jobs:
       - name: Run tests
         run: npm test
 
+  esm_cjs:
+    name: Types & Entrypoints
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: npm
+      - name: Install dependencies
+        run: npm ci
+      - name: Verify types & entrypoints
+        run: npx --yes @arethetypeswrong/cli --pack .
```

Comment:
Using `attw` isn't a direct check of `require("tough-cookie")` and `import("tough-cookie")` in node. But I guess neither is vitest, since it has its extra layer on top for mocking. Is this good enough? 🤷

Reply:
I suppose I'm going off the fact that `tsup` **should** produce valid ESM and CJS artifacts and `attw` **should** verifies if your types and entrypoints are configured correctly. Maybe it's not a true test since the `require` and `import` are not verified directly but it seems reasonable to trust that these tools are working as advertised.

Setting up a full integration test harness just to verify the `require` and `import` seems a bit heavy. This is why I tried verifying locally then setting up some of the above checks as a safe guard.

How would you prefer we test this @wjhsf?